sendmail.py

```diff
@@ -2,19 +2,33 @@
 from email.message import EmailMessage
 from decouple import config
 
+
 def send(subject, recipient, body):
+    # was older gmail variables set?
     gmail_user = config('GMAIL_USER')
     gmail_password = config('GMAIL_PWD')
```

Comment:
The readme and config file no longer have GMAIL_USER and GMAIL_PWD so I would recommend dropping these.